package.json

```diff
@@ -24,6 +24,7 @@
     "primereact": "^8.1.1",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
+    "react-grid-system": "^8.1.5",
```

Comment:
Also; pls check it out `primeflex` for layout, grid & typography. It is css-driven which is preferable/efficient, IMO; rather than composing as a component on those presentation only aspect, etc.

- https://www.primefaces.org/primeflex/
- https://www.google.com/search?q=primeflex  (YouTube)

If any help needing/blocker, we can reckon with their Forum/Discord link there, etc...

Reply:
Yup!

Reply:
Sure! Replacing it to `primeflex`